### PR TITLE
Fix bsc#1131899

### PR DESCRIPTION
**Pull request by dmpop**
Fix https://bugzilla.suse.com/show_bug.cgi?id=1131899

### CODE REVIEW

File: xml/installation-installation-ses_integration.xml [hunk 1]
```diff
@@ -376,4 +376,49 @@ rgw keystone verify ssl = false # If keystone is using self-signed
...
    </procedure>
   </section>
+    <section>
+   <title>Enable Copy-On-Write Cloning of Images</title>
+   <para>
+    Due to a security issue described in <link
+    xlink:href="http://docs.ceph.com/docs/master/rbd/rbd-openstack/?highlight=uuid#enable-copy-on-write-cloning-of-images"
+    />, the copy-on-write cloning of images when
+    &o_img; and &o_blockstore; are both using a &ceph; back-end is not
+    supported. However, if you want to use this feature for faster operation,
```
Comment (asettle) [inline xml/installation-installation-ses_integration.xml]: &o_img; and &o_blockstore; are both using a &ceph;  back-end is not supported. <- this doesn't make sense? Is there something missing here?

Comment (dmpop) [reply xml/installation-installation-ses_integration.xml]: I dunno. That's what the description said. Maybe @bryanstephenson can shed some light on it.

Comment (csymons-suse) [reply xml/installation-installation-ses_integration.xml]: Due to a security issue described in ... , the COW cloning of images is not supported when Glance and Cinder are both using a Ceph back-end.

Comment (asettle) [reply xml/installation-installation-ses_integration.xml]: Makes sense to me, thanks Carl.

**Comment by asettle** [comment]
@dmpop is out for the next few days and has given me permission to update this PR as required.

**Comment by asettle** [comment]
Last review required from @bryanstephenson :)

File: xml/installation-installation-ses_integration.xml (hunk 1 above)
Comment (bryanstephenson) [reply xml/installation-installation-ses_integration.xml]: It makes sense to me. Maybe say it this way instead:

the copy-on-write cloning of images when both
    &o_img; and &o_blockstore; are using a &ceph; back-end is not
    supported.

**Comment by asettle** [comment]
Merging as @dmpop is away :)

**Pull request merged**